src/middlewared/middlewared/plugins/vm.py

```diff
@@ -394,7 +403,7 @@ def flags(self):
         Bool('autostart'),
         register=True,
         ))
-    async def do_create(self, data):
+    async def create(self, data):
```

Comment:
No, it should stay as do_create, you just dont call vm.do_create, you call vm.create